exercises/roman-numerals/test/Tests.hs

```diff
@@ -17,66 +17,85 @@ specs = describe "roman-numerals" $
 
     test Case{..} = it explanation assertion
       where
-        explanation = show number
+        explanation = show number ++ ": " ++ description
```

Comment:
Yeah! The `27`'s description is confusing...

I still prefer the [assertEqual solution](https://github.com/exercism/xhaskell/issues/517), instead of changing the test's `description`, because it is shown only when it fails.

Anyway, you solution definitely solves the problem here! 👍